examples/pip_parse_vendored/requirements.bzl

```diff
@@ -1,7 +1,7 @@
 """Starlark representation of locked requirements.
 
 @generated by rules_python pip_parse repository rule
-from //:requirements.txt
+from @//:requirements.txt
```

Comment:
I had to do update this to get CI to pass

Reply:
I believe it is related to Bazel 6 https://github.com/bazelbuild/rules_python/pull/938

Reply:
@groodt yep

Reply:
That is spooky that our CI uses a floating version of bazel rather than what's pinned in our .bazelversion file - I think we ought to fix that along with updating that golden file.